---
resume: true
title: Will Hannon
email: hannonww@gmail.com
linkedin: https://www.linkedin.com/in/williamhannon/
github: https://github.com/WillHannon-MCB
---

## Summary

I have nearly 10 years of experience in biomedical research and computational biology. I develop software tools to help biologists explore complex biological data, create websites and interactive visualizations that turn results into resources, and build reproducible pipelines for analyzing NGS and DMS datasets. I’m seeking opportunities to apply my skills in a cross-functional team doing cutting-edge research.

## Skills

**Languages:** `Python` • `Javascript` • `R` • `Bash`  
**Libraries:** Visualization with `Altair/Vega`, `D3.js`, and `ggplot2` • Analysis with `pandas`, `numpy`, and `tidyverse` • Web development with `Vue.js` and `VitePress`  
**Tools:** `Git` • `Conda` • `Snakemake`  
**Scientific:** Viral Evolution • Deep Mutational Scanning • Next-generation Sequencing 
**Professional:** Collaboration • Teaching • Mentorship • Communication  

## Education

**PhD, Molecular and Cellular Biology | University of Washington | Seattle, WA | December 2023**  
*[Thesis work](https://github.com/WillHannon-MCB/Thesis) developing computational approaches to study the dynamics of viral evolution and pathogenesis from high-throughput datasets advised by [Jesse Bloom](https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html)*

**BA, Biochemistry | Case Western Reserve University | Cleveland, OH | May 2017**  
*cum laude*

## Work Experience

**Fred Hutchinson Cancer Research Center | January 2023 to Present**  
*Data Scientist*  

I work with a team of experimental biologists, helping them analyze, interpret, and share the results of high-throughput sequencing experiments and deep mutational scans of viral proteins.
- Our projects are constantly evolving, so I create and maintain automated analysis pipelines to quickly turn new data into actionable results.
- We need biological context to interpret our results, so I develop web-based software tools for visualizing large mutation-based datasets on interactive protein structures.
- Our findings provide valuable information for public health, therapeutic development, and basic virology, so I build online resources like websites and interactive visualizations to make them accessible to the scientific community. For example, I create modular websites with interactive tools for exploring DMS data, such as [this site](https://dms-vep.org/Flu_H5_American-Wigeon_South-Carolina_2021-H5N1_DMS/) showing the impact of mutations in avian influenza H5.

**University of Washington | August 2019 to December 2023**  
*Graduate Student––Advised by [Jesse Bloom](https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html)*

I studied the dynamics of viral evolution using large datasets and high-throughput experiments, culminating in three first-author publications and contributions to six other publications.
- Determined how transmission shapes viral evolution in SARS-CoV-2 outbreaks using a combination of deep-sequencing data and statistical modeling.
- Shed light on the evolutionary trajectory that enables measles to escalate from an acute respiratory infection into a chronic neurological disorder using phylogenetic methods and haplotyping.
- Created a web-based tool called [`dms-viz`](https://dms-viz.github.io/v0/) to help scientists visualize mutation-associated data in the context of interactive 3D protein structures.

**National Institutes of Health | June 2017 to June 2019**  
*Postbaccalaureate CRTA Fellow––Advised by [Philipp Oberdoerffer](https://profiles.hopkinsmedicine.org/provider/philipp-oberdoerffer/2777574) and [Pedro Batista](https://ccr.cancer.gov/staff-directory/pedro-j-batista)*

I studied the impact of RNA modifications—particularly N6-methyladenosine (m6A)—on genome maintenance after DNA Double-Strand Breaks.
- Developed an automated image analysis pipeline with Python to count fluorescent foci, providing quantitative insights into DNA damage response mechanisms.
- Piloted a new immunoprecipitation and sequencing technique from literature to investigate the role of RNA-DNA hybrids in DNA damage response, expanding the lab's research directions.
- Gained experience in NGS library prep, confocal fluorescence microscopy, and tissue culture.

**Case Western Reserve University | November 2016 to May 2017**  
*Research Assistant––Advised by [Jeff Coller](https://profiles.hopkinsmedicine.org/provider/jeff-coller/2777263)*

I studied how codon optimality influences mRNA stability and protein expression.
- Developed a computational pipeline in R to analyze the relationship between influenza codon usage, RNA stability, and pathogenicity over time.

## Selected Publications

1. **Hannon, WW**, Bloom JD. `dms-viz`: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets. Journal of Open Source Software (2024)
2. Yousaf I\*, **Hannon WW**\*, Donohue RC, Pfaller CK, Yadav K, Dikdan RJ, Tyagi S, Schroeder DC, Shieh WJ, Rota PA, Feder AF, Cattaneo R. Spatial dynamics and evolution of a measles virus genome collective that drove subacute sclerosing panencephalitis. PLOS Pathogens (2023)
3. **Hannon WW**, Roychoudhury P, Xie H, Shrestha L, Addetia A, Jerome KR, Greninger AL, Bloom JD. Narrow transmission bottlenecks and limited within-host viral diversity during a SARS-CoV-2 outbreak on a fishing boat. Virus Evolution (2022)
4. Dadonaite B, Crawford KHD, Radford CE, Farrell AG, Yu CT, **Hannon WW**, Zhou P, Andrabi R, Burton DR, Liu L. A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike. Cell (2023)
5. Starr TN\*, Greaney AJ\*, **Hannon WW**, Loes AN, Hauser K, Dillen JR, Ferri E, Farrell AG, Dadonaite B, McCallum M. Shifting mutational constraints in the SARS-CoV-2 receptor-binding domain during viral evolution. Science (2022)
6. Starr TN, Greaney AJ, Addetia A, **Hannon WW**, Choudhary MC, Dingens AS, Li JZ, Bloom JD. Prospective mapping of viral mutations that escape antibodies used to treat COVID-19. Science (2021)

## Teaching and Mentorship

**Research Mentor | Bloom Lab, Fred Hutchinson Cancer Center | 2021 to Present**  
Mentored 2 rotation students (10 weeks each) and 1 undergraduate student (2 months) on independent computational research projects.  
**Teaching Assistant | Department of Biochemistry, University of Washington | Winter 2021**  
Biochemistry (BIOC 441): Led two weekly quiz sections of 30+ students, and assisted in curriculum and exam writing.  
**Teaching Assistant | Department of Molecular and Cellular Biology, University of Washington | Fall 2020**  
Tools for Computational Biology (MCB 517): Taught incoming graduate students basic computational skills.  
