---
resume: true
title: Will Hannon
email: hannonww@gmail.com
linkedin: https://www.linkedin.com/in/williamhannon/
github: https://github.com/WillHannon-MCB
---

## Professional Summary

Driven and diligent computational biologist with experience deriving insights from large-scale datasets and high-throughput experiments. Expertise in developing robust computational tools, performing in-depth analyses of sequencing data, and working in interdisciplinary, collaborative settings to solve complex biological problems. Seeking to apply data science and computational biology expertise to advance innovative research projects.

## Skills

**Languages:** `Python` • `Javascript` • `R` • `Bash`  
**Libraries:** Visualization with `Altair/Vega`, `D3.js`, and `ggplot2` • Analysis with `pandas`, `numpy`, and `tidyverse` • Web development with `Vue.js` and `VitePress`  
**Tools:** `Git` • `Conda` • `Snakemake`  
**Scientific:** Virology • High-throughput sequencing analysis • Bioinformatics-informed experimental design  
**Professional:** Collaboration • Teaching • Mentorship  

## Education

**PhD, Molecular and Cellular Biology | University of Washington | Seattle, WA | December 2023**  
*[Thesis work](https://github.com/WillHannon-MCB/Thesis) developing computational approaches to study the dynamics of viral evolution and pathogenesis from high-throughput datasets advised by [Jesse Bloom](https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html)*

**BA, Biochemistry | Case Western Reserve University | Cleveland, OH | May 2017**  
*cum laude*

## Work Experience

**Fred Hutchinson Cancer Research Center | January 2023 to Present**  
*Data Scientist*  

- [Developed infrastructure](https://dms-vep.org/) for sharing deep mutational scanning experiment results, complete with detailed documentation and interactive visualizations, enabling broader scientific engagement.
- Led the redesign of the [lab’s website](https://jbloomlab.org/), centralizing important resources such as software, blog posts, and publications for improved accessibility and organization.
- Collaborated with wet-lab biologists, providing data analysis support, experimental design guidance, and [training lab members in computational tools and bioinformatics skills](https://jbloomlab.github.io/bench-to-byte/).

**University of Washington | August 2019 to December 2023**  
*Graduate Research Assistant––Advised by [Jesse Bloom](https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html)*

- Designed and implemented reusable computational pipelines to analyze viral deep-sequencing data, contributing to our understanding of SARS-CoV-2 transmission dynamics.
- Applied phylogenetic techniques and computational modeling to elucidate how measles evolves from an acute respiratory infection to a chronic neurological disorder.
- Created a [web-based tool to visualize mutation-associated data](https://dms-viz.github.io/v0/) within a 3D protein structure, improving researchers' ability to interpret structural impacts of mutations.

**National Institutes of Health | June 2017 to June 2019**  
*Post-baccalaureate Fellow––Advised by [Philipp Oberdoerffer](https://profiles.hopkinsmedicine.org/provider/philipp-oberdoerffer/2777574) and [Pedro Batista](https://ccr.cancer.gov/staff-directory/pedro-j-batista)*

- Self-taught Python and developed an automated image analysis pipeline to count fluorescent foci, providing quantitative insights into DNA damage response mechanisms.
- Pioneered new experiments investigating the role of RNA-DNA hybrids in DNA damage response, expanding the lab's research directions.
- Researched the impact of RNA modifications, particularly N6-methyladenosine (m6A), on genome stability during DNA Double-Strand Breaks, advancing knowledge in genome maintenance mechanisms.

**Case Western Reserve University | November 2016 to May 2017**  
*Research Assistant––Advised by [Jeff Coller](https://profiles.hopkinsmedicine.org/provider/jeff-coller/2777263)*

- Developed a computational pipeline to analyze the relationship between influenza codon usage, RNA stability, and pathogenicity, enhancing understanding of viral evolution and adaptation mechanisms.

## Selected Publications

1. `dms-viz`: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets; **Hannon, William W**; Bloom, Jesse D; JOSS, *2024*  
2. Spatial dynamics and evolution of a measles virus genome collective that drove subacute sclerosing panencephalitis; Yousaf, Iris*; **Hannon, William W**\*, Donohue, Ryan C; Pfaller, Christian K; Yadav, Kalpana; Dikdan, Ryan J; Tyagi, Sanjay; Schroeder, Declan C; Shieh, Wun-Ju; Rota, Paul A; Feder, Alison F; Cattaneo, Roberto; PLOS Pathogens, *2023*  
3. Narrow transmission bottlenecks and limited within-host viral diversity during a SARS-CoV-2 outbreak on a fishing boat; **Hannon, William W**; Roychoudhury, Pavitra; Xie, Hong; Shrestha, Lasata; Addetia, Amin; Jerome, Keith R; Greninger, Alexander L; Bloom, Jesse D; Virus Evolution, *2022*  
4. A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike: Dadonaite, Bernadeta; Crawford, Katharine HD; Radford, Caelan E; Farrell, Ariana G; Timothy, C Yu; **Hannon, William W**; Zhou, Panpan; Andrabi, Raiees; Burton, Dennis R; Liu, Lihong; Cell, *2023*  
5. Shifting mutational constraints in the SARS-CoV-2 receptor-binding domain during viral evolution; Starr, Tyler N\*; Greaney, Allison J\*; **Hannon, William W**; Loes, Andrea N; Hauser, Kevin; Dillen, Josh R; Ferri, Elena; Farrell, Ariana Ghez; Dadonaite, Bernadeta; McCallum, Matthew;  Science, *2022*  
6. Prospective mapping of viral mutations that escape antibodies used to treat COVID-19; Starr, Tyler N; Greaney, Allison J; Addetia, Amin; **Hannon, William W**; Choudhary, Manish C; Dingens, Adam S; Li, Jonathan Z; Bloom, Jesse D; Science, *2021*  

## Teaching and Mentorship

**Research Mentor | Bloom Lab, Fred Hutchinson Cancer Center | 2021 to Present**  
Mentored 2 rotation students (10 weeks each) and 1 undergraduate student (2 months) on independent computational research projects.  
**Teaching Assistant | Department of Biochemistry, University of Washington | Winter 2021**  
Biochemistry (BIOC 441): Led two weekly quiz sections of 30+ students, and assisted in curriculum and exam writing.  
**Teaching Assistant | Department of Molecular and Cellular Biology, University of Washington | Fall 2020**  
Tools for Computational Biology (MCB 517): Taught incoming graduate students basic computational skills.  
