import{_ as a,c as n,e as i,o as t}from"./chunks/framework.CmXP7DcN.js";const u=JSON.parse('{"title":"Will Hannon","description":"","frontmatter":{"resume":true,"title":"Will Hannon","email":"hannonww@gmail.com","linkedin":"https://www.linkedin.com/in/williamhannon/","github":"https://github.com/WillHannon-MCB"},"headers":[],"relativePath":"resume.md","filePath":"resume.md"}'),o={name:"resume.md"};function r(s,e,l,d,c,h){return t(),n("div",null,e[0]||(e[0]=[i('<h2 id="professional-summary" tabindex="-1">Professional Summary <a class="header-anchor" href="#professional-summary" aria-label="Permalink to &quot;Professional Summary&quot;">​</a></h2><p>Driven and diligent computational biologist with experience deriving insights from large-scale datasets and high-throughput experiments. Expertise in developing robust computational tools, performing in-depth analyses of sequencing data, and working in interdisciplinary, collaborative settings to solve complex biological problems. Seeking to apply data science and computational biology expertise to advance innovative research projects.</p><h2 id="skills" tabindex="-1">Skills <a class="header-anchor" href="#skills" aria-label="Permalink to &quot;Skills&quot;">​</a></h2><p><strong>Languages:</strong> <code>Python</code> • <code>Javascript</code> • <code>R</code> • <code>Bash</code><br><strong>Libraries:</strong> Visualization with <code>Altair/Vega</code>, <code>D3.js</code>, and <code>ggplot2</code> • Analysis with <code>pandas</code>, <code>numpy</code>, and <code>tidyverse</code> • Web development with <code>Vue.js</code> and <code>VitePress</code><br><strong>Tools:</strong> <code>Git</code> • <code>Conda</code> • <code>Snakemake</code><br><strong>Scientific:</strong> Virology • High-throughput sequencing analysis • Bioinformatics-informed experimental design<br><strong>Professional:</strong> Collaboration • Teaching • Mentorship</p><h2 id="education" tabindex="-1">Education <a class="header-anchor" href="#education" aria-label="Permalink to &quot;Education&quot;">​</a></h2><p><strong>PhD, Molecular and Cellular Biology | University of Washington | Seattle, WA | December 2023</strong><br><em><a href="https://github.com/WillHannon-MCB/Thesis" target="_blank" rel="noreferrer">Thesis work</a> developing computational approaches to study the dynamics of viral evolution and pathogenesis from high-throughput datasets advised by <a href="https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html" target="_blank" rel="noreferrer">Jesse Bloom</a></em></p><p><strong>BA, Biochemistry | Case Western Reserve University | Cleveland, OH | May 2017</strong><br><em>cum laude</em></p><h2 id="work-experience" tabindex="-1">Work Experience <a class="header-anchor" href="#work-experience" aria-label="Permalink to &quot;Work Experience&quot;">​</a></h2><p><strong>Fred Hutchinson Cancer Research Center | January 2023 to Present</strong><br><em>Data Scientist</em></p><ul><li><a href="https://dms-vep.org/" target="_blank" rel="noreferrer">Developed infrastructure</a> for sharing deep mutational scanning experiment results, complete with detailed documentation and interactive visualizations, enabling broader scientific engagement.</li><li>Led the redesign of the <a href="https://jbloomlab.org/" target="_blank" rel="noreferrer">lab’s website</a>, centralizing important resources such as software, blog posts, and publications for improved accessibility and organization.</li><li>Collaborated with wet-lab biologists, providing data analysis support, experimental design guidance, and <a href="https://jbloomlab.github.io/bench-to-byte/" target="_blank" rel="noreferrer">training lab members in computational tools and bioinformatics skills</a>.</li></ul><p><strong>University of Washington | August 2019 to December 2023</strong><br><em>Graduate Research Assistant––Advised by <a href="https://www.fredhutch.org/en/faculty-lab-directory/bloom-jesse.html" target="_blank" rel="noreferrer">Jesse Bloom</a></em></p><ul><li>Designed and implemented reusable computational pipelines to analyze viral deep-sequencing data, contributing to our understanding of SARS-CoV-2 transmission dynamics.</li><li>Applied phylogenetic techniques and computational modeling to elucidate how measles evolves from an acute respiratory infection to a chronic neurological disorder.</li><li>Created a <a href="https://dms-viz.github.io/v0/" target="_blank" rel="noreferrer">web-based tool to visualize mutation-associated data</a> within a 3D protein structure, improving researchers&#39; ability to interpret structural impacts of mutations.</li></ul><p><strong>National Institutes of Health | June 2017 to June 2019</strong><br><em>Post-baccalaureate Fellow––Advised by <a href="https://profiles.hopkinsmedicine.org/provider/philipp-oberdoerffer/2777574" target="_blank" rel="noreferrer">Philipp Oberdoerffer</a> and <a href="https://ccr.cancer.gov/staff-directory/pedro-j-batista" target="_blank" rel="noreferrer">Pedro Batista</a></em></p><ul><li>Self-taught Python and developed an automated image analysis pipeline to count fluorescent foci, providing quantitative insights into DNA damage response mechanisms.</li><li>Pioneered new experiments investigating the role of RNA-DNA hybrids in DNA damage response, expanding the lab&#39;s research directions.</li><li>Researched the impact of RNA modifications, particularly N6-methyladenosine (m6A), on genome stability during DNA Double-Strand Breaks, advancing knowledge in genome maintenance mechanisms.</li></ul><p><strong>Case Western Reserve University | November 2016 to May 2017</strong><br><em>Research Assistant––Advised by <a href="https://profiles.hopkinsmedicine.org/provider/jeff-coller/2777263" target="_blank" rel="noreferrer">Jeff Coller</a></em></p><ul><li>Developed a computational pipeline to analyze the relationship between influenza codon usage, RNA stability, and pathogenicity, enhancing understanding of viral evolution and adaptation mechanisms.</li></ul><h2 id="selected-publications" tabindex="-1">Selected Publications <a class="header-anchor" href="#selected-publications" aria-label="Permalink to &quot;Selected Publications&quot;">​</a></h2><ol><li><code>dms-viz</code>: Structure-informed visualizations for deep mutational scanning and other mutation-based datasets; <strong>Hannon, William W</strong>; Bloom, Jesse D; JOSS, <em>2024</em></li><li>Spatial dynamics and evolution of a measles virus genome collective that drove subacute sclerosing panencephalitis; Yousaf, Iris*; <strong>Hannon, William W</strong>*, Donohue, Ryan C; Pfaller, Christian K; Yadav, Kalpana; Dikdan, Ryan J; Tyagi, Sanjay; Schroeder, Declan C; Shieh, Wun-Ju; Rota, Paul A; Feder, Alison F; Cattaneo, Roberto; PLOS Pathogens, <em>2023</em></li><li>Narrow transmission bottlenecks and limited within-host viral diversity during a SARS-CoV-2 outbreak on a fishing boat; <strong>Hannon, William W</strong>; Roychoudhury, Pavitra; Xie, Hong; Shrestha, Lasata; Addetia, Amin; Jerome, Keith R; Greninger, Alexander L; Bloom, Jesse D; Virus Evolution, <em>2022</em></li><li>A pseudovirus system enables deep mutational scanning of the full SARS-CoV-2 spike: Dadonaite, Bernadeta; Crawford, Katharine HD; Radford, Caelan E; Farrell, Ariana G; Timothy, C Yu; <strong>Hannon, William W</strong>; Zhou, Panpan; Andrabi, Raiees; Burton, Dennis R; Liu, Lihong; Cell, <em>2023</em></li><li>Shifting mutational constraints in the SARS-CoV-2 receptor-binding domain during viral evolution; Starr, Tyler N*; Greaney, Allison J*; <strong>Hannon, William W</strong>; Loes, Andrea N; Hauser, Kevin; Dillen, Josh R; Ferri, Elena; Farrell, Ariana Ghez; Dadonaite, Bernadeta; McCallum, Matthew; Science, <em>2022</em></li><li>Prospective mapping of viral mutations that escape antibodies used to treat COVID-19; Starr, Tyler N; Greaney, Allison J; Addetia, Amin; <strong>Hannon, William W</strong>; Choudhary, Manish C; Dingens, Adam S; Li, Jonathan Z; Bloom, Jesse D; Science, <em>2021</em></li></ol><h2 id="teaching-and-mentorship" tabindex="-1">Teaching and Mentorship <a class="header-anchor" href="#teaching-and-mentorship" aria-label="Permalink to &quot;Teaching and Mentorship&quot;">​</a></h2><p><strong>Research Mentor | Bloom Lab, Fred Hutchinson Cancer Center | 2021 to Present</strong><br> Mentored 2 rotation students (10 weeks each) and 1 undergraduate student (2 months) on independent computational research projects.<br><strong>Teaching Assistant | Department of Biochemistry, University of Washington | Winter 2021</strong><br> Biochemistry (BIOC 441): Led two weekly quiz sections of 30+ students, and assisted in curriculum and exam writing.<br><strong>Teaching Assistant | Department of Molecular and Cellular Biology, University of Washington | Fall 2020</strong><br> Tools for Computational Biology (MCB 517): Taught incoming graduate students basic computational skills.</p>',20)]))}const g=a(o,[["render",r]]);export{u as __pageData,g as default};
